arm-datalake-analytics/job/2016-11-01/swagger/job.json

```diff
@@ -0,0 +1,1070 @@
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "the completion time of the job."
+        },
+        "state": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the job state. When the job is in the Ended state, refer to Result and ErrorMessage for details.",
+          "enum": [
+            "Accepted",
+            "Compiling",
+            "Ended",
+            "New",
+            "Queued",
+            "Running",
+            "Scheduling",
+            "Starting",
+            "Paused",
+            "WaitingForCapacity"
+          ],
+          "x-ms-enum": {
+            "name": "JobState",
+            "modelAsString": false
+          }
+        },
+        "result": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the result of job execution or the current result of the running job.",
+          "enum": [
+            "None",
+            "Succeeded",
+            "Cancelled",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "JobResult",
+            "modelAsString": false
+          }
+        },
+        "logFolder": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the log folder path to use in the following format: adl://<accountName>.azuredatalakestore.net/system/jobservice/jobs/Usql/2016/03/13/17/18/5fe51957-93bc-4de0-8ddc-c5a4753b068b/logs/."
+        },
+        "logFilePatterns": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },        
+          "description": "the list of log file name patterns to find in the logFolder. '*' is the only matching character allowed. Example format: jobExecution*.log or *mylog*.txt"
+        },
+        "stateAuditRecords": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/JobStateAuditRecord"
+          },
+          "description": "the job state audit records, indicating when various operations have been performed on this job."
+        },
+        "properties": {
+          "$ref": "#/definitions/JobProperties",
+          "description": "the job specific properties."
+        }
+      },
+      "required": [
+        "name",
+        "type",
+        "properties"
+      ],
+      "description": "The common Data Lake Analytics job information properties."
+    },
+    "JobInfoListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/JobInformation"
+          },
+          "description": "the list of jobInfo items."
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the link (url) to the next page of results."
+        },
+        "count": {
+          "type": "integer",
```

Comment:
please make sure to do so in all the models that are returned in a pageable result.